src/main/groovy/org/sharegov/mdcgis/PropertyInfoService.groovy

```diff
@@ -207,39 +214,42 @@ class PropertyInfoService {
 		if(!dataFromLayers)
 			return propertyType
 		
-		
-		if (dataFromLayers['MDC.PTXGIS']){
-			String propertyTypeFlag = 	dataFromLayers['MDC.PTXGIS'].attributes?.CONDO?:dataFromLayers['MDC.PTXGIS'].CONDO
-			switch (propertyTypeFlag){
-				case 'C':
-					propertyType = PropertyType.CONDO
-					break
-				case 'P':
-					propertyType = PropertyType.MULTI
-					break
-				case 'N':
-					propertyType = PropertyType.UNDEFINED
-					break
-				default:
-					propertyType = PropertyType.UNDEFINED
-
-			}
+		if (dataFromLayers['MDC.PaGIS']){
+			propertyType = getPropertyType(dataFromLayers, 'MDC.PaGIS')
 		}
-		else if (dataFromLayers['MDC.Parcels']) {
-			String propertyTypeFlag = 	dataFromLayers['MDC.Parcels'].attributes?.CONDO?:dataFromLayers['MDC.Parcels'].CONDO
-			switch (propertyTypeFlag){
-				case 'C':
-					propertyType = PropertyType.MULTI
-					break
-				default:
-					propertyType = PropertyType.UNDEFINED
-			}
+		else if (dataFromLayers['MDC.PaParcel']) {
+			propertyType = getPropertyType(dataFromLayers, 'MDC.PaParcel')
 		}
 
 		return propertyType
 	}
 
 	/**
+	 * Get Property type.
+	 *
```

Comment:
I would have a lengthy explanation on how you calculate each value, like for example MULTI